### PR TITLE
layers: Cleanup of UNASSIGNED VUIDs

### PR DESCRIPTION
**Pull request by spencer-lunarg**
Created some Spec issues to remove more, but some UNASSIGNED VUIDs actually had a proper VUID to give it

### CODE REVIEW

File: layers/core_checks/synchronization_validation.cpp [hunk 1]
```diff
@@ -2070,20 +2070,18 @@ class ValidatorState {
...
     static bool ValidateAtQueueSubmit(const QUEUE_STATE *queue_state, const ValidationStateTracker *device_data,
                                       uint32_t src_family, uint32_t dst_family, const ValidatorState &val) {
-        auto error_code = QueueError::kSubmitQueueMustMatchSrcOrDst;
         uint32_t queue_family = queue_state->queueFamilyIndex;
         if ((src_family != queue_family) && (dst_family != queue_family)) {
-            const std::string val_code = GetBarrierQueueVUID(val.loc_, error_code);
             const char *src_annotation = val.GetFamilyAnnotation(src_family);
             const char *dst_annotation = val.GetFamilyAnnotation(dst_family);
-            return device_data->LogError(queue_state->Handle(), val_code,
+            return device_data->LogError(queue_state->Handle(), "VUID-vkQueueSubmit-pSubmits-04626",
```
Comment (spencer-lunarg) [inline layers/core_checks/synchronization_validation.cpp:2077]: This was discussed in https://gitlab.khronos.org/vulkan/vulkan/-/merge_requests/5799 that this is the correct VUID

File: layers/core_checks/shader_validation.cpp [hunk 2]
```diff
@@ -1508,29 +1508,48 @@ bool CoreChecks::ValidateCooperativeMatrix(const SHADER_MODULE_STATE &module_sta
...
+                        skip |= LogError(module_state.vk_shader_module(), "VUID-RuntimeSpirv-OpCooperativeMatrixMulAddNV-06318",
+                                         "OpCooperativeMatrixMulAddNV (result id = %u) operands don't match a supported matrix "
+                                         "VkCooperativeMatrixPropertiesNV",
+                                         insn.Word(2));
+                    } else if (!valid_b) {
+                        skip |= LogError(module_state.vk_shader_module(), "VUID-RuntimeSpirv-OpCooperativeMatrixMulAddNV-06319",
+                                         "OpCooperativeMatrixMulAddNV (result id = %u) operands don't match a supported matrix "
+                                         "VkCooperativeMatrixPropertiesNV",
+                                         insn.Word(2));
+                    } else if (!valid_b) {
```
Comment (arno-lunarg) [inline layers/core_checks/shader_validation.cpp:1551]:
```suggestion
                    } else if (!valid_d) {
```

**Review by arno-lunarg — APPROVED** [review]
LGTM! I just spotted some typos

File: layers/error_message/validation_error_enums.h [hunk 3]
```diff
@@ -39,38 +38,26 @@
...
-[[maybe_unused]] static const char *kVUID_Core_Swapchain_PreTransform = "UNASSIGNED-CoreValidation-SwapchainPreTransform";
 [[maybe_unused]] static const char *kVUID_Core_BindImageMemory_Swapchain = "UNASSIGNED-CoreValidation-BindImageMemory-Swapchain";
 
-[[maybe_unused]] static const char *kVUID_Core_Image_InvalidFormatLimitsViolation = "UNASSIGNED-CoreValidation-Image-InvalidFormatLimitsViolation";
-
 [[maybe_unused]] static const char *kVUID_Core_PushDescriptorUpdate_TemplateType = "UNASSIGNED-CoreValidation-vkCmdPushDescriptorSetWithTemplateKHR-descriptorUpdateTemplate-templateType";
 [[maybe_unused]] static const char *kVUID_Core_PushDescriptorUpdate_Template_SetMismatched = "UNASSIGNED-CoreValidation-vkCmdPushDescriptorSetWithTemplateKHR-set";
 [[maybe_unused]] static const char *kVUID_Core_PushDescriptorUpdate_Template_LayoutMismatched = "UNASSIGNED-CoreValidation-vkCmdPushDescriptorSetWithTemplateKHR-layout";
 
+// TODO - Need to be moved to Best Practice
```
[the remark this answers is not in the document]
Comment (spencer-lunarg) [reply layers/error_message/validation_error_enums.h:53]: https://github.com/orgs/KhronosGroup/projects/12?pane=issue&itemId=23676891

**Pull request merged**